sktime/transformations/hierarchical/aggregate.py

```diff
@@ -63,97 +64,105 @@ def _transform(self, X, y=None):
 
         Parameters
         ----------
-        X : Panel of mtype pd_multiindex_hier
-            Data to be transformed
+        X : Series or Panel of mtype X_inner_mtype
```

Comment:
I guess you want to say `X : pd.DataFrame`, with any number of row levels and not nested?